packages/vue/tests/unit/AdvancedImage.spec.ts

```diff
@@ -80,8 +80,6 @@ describe("AdvancedImage.vue", () => {
 
     // trigger AdvancedImage.onUpdate
     await component.setProps({
-      cldImg: cloudinaryImage,
-      plugins: [mock],
```

Comment:
These two are not needed to trigger render and for some reason were causing a typescript issue in Travis CI.